conf.d/main

```diff
@@ -52,9 +52,7 @@ chown -R node:node $NODEAPP
 su - node -c "cd $NODEAPP && NODE_ENV=production ./bin/run.sh >$NODEAPP/log.tmp 2>&1 &"
 
 echo "Waiting for Etherpad to complete first run."
-while ! grep "You can access your Etherpad instance" $NODEAPP/log.tmp; do
-    sleep 10
-done
```

Comment:
IIRC this step was required at some point for building successfully. I'm pretty sure it was there to ensure that Etherpad starts cleanly (before being killed) when doing the builds. If there's another better way to ensure that Etherpad has started cleanly, then I'm open to supporting that. But I think that this is probably still worth having (assuming that it does still work, now you've implemented the `argon2` stuff). If it still doesn't work, check the logs (perhaps the text has changed and no longer matches `You can access your Etherpad instance`?).

I'm assuming that it built cleanly for you without this, which suggests that a `10` second sleep might be overkill. Feel free to reduce that (even to `1` if you want - it's in a loop so it will still wait).